Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.10-alpine
+
+WORKDIR /src
+
+COPY . . 
+
+RUN apk add --no-cache build-base
+RUN pip install --no-cache-dir -r requirements.txt
+
+CMD [ "python", "udemy_enroller.py", "--max-pages=900000"]
```

Comment:
There will be no settings configured so this will hang forever

Reply:
Hello, it is enough if I add a description to the README.md about how to use the image? (I don't want that the image is built with the credentials included), instead you can mount your `settings.yaml` as a volume/file with `-v$(pwd)/settings.yaml:/root/.udemy_enroller/settings.yaml` or start the image in interactive mode and enter your credentials.

Reply:
Ye that would be ok. 
Would it be possible to pass in the variables for execution as well? Having max-pages at 900000 is overkill and in order to run each time you need to restart the container instead of say using a make file which calls the container interactively